contrib/TestHarness2/test_harness/run.py

```diff
@@ -574,12 +574,17 @@ def run_tests(
             result = result and run.success
             test_picker.add_time(test_files[0], run.run_time, run.summary.out)
             decorate_summary(run.summary.out, file, seed + count, run.buggify_enabled)
-            if unseed_check and run.summary.unseed:
+            if unseed_check and run.summary.unseed >= 0:
```

Comment:
I suspect you'll need an analogous change in TestHarness, since I think it is still in use and may be what CI uses

Reply:
I could be wrong about that, but we should confirm that the original TestHarness is not active anymore if we decide not to update it.

Reply:
@jzhou77 says that TestHarness is not in use on main now, so I think this is ok then